cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2033,4 +2033,13 @@ public static final class ArtifactCache {
     public static final String BOSS_THREADS = "artifact.cache.boss.threads";
     public static final String WORKER_THREADS = "artifact.cache.worker.threads";
   }
+
+  /**
+   * Constants for MetadataConsumer
+   */
+  public static final class MetadataConsumer {
+    public static final String PROJECT_NAME = "metadata.consumer.project.name";
```

Comment:
If it is GCP specific, ideally they shouldn't be in cdap-common. Also, if it is used only by another repo, when not defining the constants there instead?

Reply:
Ah okay, so this will also be used in the next PR(wip), something like - https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/internal/events/ProgramStatusEventPublisher.java#L88

Reply:
Removing these constants from here now